src/types.ts

```diff
@@ -1,8 +1,6 @@
-import type { Language } from "./tokenizer/languages";
-import type { Hooks } from "./methods/hooks";
-import type { TokenizerConfig } from "./tokenizer";
-import type { IIntersectTokenScores } from "./utils";
-import type { Node } from "./radix-tree/node";
+import { IIntersectTokenScores, Language, TokenizerConfig, TokenScore } from "@lyrasearch/components";
```

Comment:
`@lyrasearch/lyra` should always provide tree-shakable default algorithms and configurations; we should not rely on external dependencies. Components will override the default algorithms carried by Lyra